components/dropdown/src/floatingUI.mjs

```diff
@@ -38,6 +69,17 @@ export default class AuroFloatingUI {
     });
   }
 
+  handleMobileFullscreen(isMobile) {
```

Comment:
This function just positions the modal without affecting size. Meaning, it doesn't do everything you might think of when you say "handleMobileFullscreen". Let's give this a more semantically correct function name. Something along the lines of `positionFullScreenBib()` or something to that affect.